content/python/concepts/functions/functions.md

```diff
@@ -42,24 +42,27 @@ doHomework()
 
 ## Function Parameters
 
-Sometimes functions require input to provide data for their code. This input is defined using parameters.
+Functions often process data to achieve their intended task. Each value required by a function is passed as an argument when that function is called.
 
-Parameters are variables that are defined in the function definition. They are assigned the values which were passed as arguments when the function was called, elsewhere in the code.
+Parameters are variables that are declared in the function definition. When a function is called, each parameter is assigned the value which was passed as a corresponding argument.
 
-For example, the function definition defines parameters for a character, a setting, and a skill, which are used as inputs to write the first sentence of a book.
+For example, the function below contains parameters for a `character`, a `setting`, and a `special_skill`, which are used as inputs to write the first sentence of a book.
 
 ```py
 def write_a_book(character, setting, special_skill):
   print(character + " is in " +
         setting + " practicing her " +
         special_skill)
+
+write_a_book("Naomi", "engineering", "welding")
+# Output: Naomi is in engineering practicing her welding
```

Comment:
Can we remove this comment and write the following below this code-snippet?

The output will be:

```
Naomi is in engineering practicing her welding
```